cudamapper/include/claragenomics/cudamapper/index.hpp

```diff
@@ -72,6 +72,14 @@ class Index
     /// \return number of reads in input data
     virtual read_id_t number_of_reads() const = 0;
 
+    /// \brief returns look up table array mapping read id to read name
+    /// \return the array mapping read id to read name
+    virtual const std::vector<std::string>& read_id_to_read_names() const = 0;
```

Comment:
Really nitpicky: shouldn't this be `read_ids_to_read_names`?

Reply:
I had factored last 's' for the whole string! :D will change.